src/commandDetails/miscellaneous/member.ts

```diff
@@ -69,7 +69,7 @@ const memberExecuteCommand: SapphireMessageExecuteType = async (
 export const memberCommandDetails: CodeyCommandDetails = {
   name: 'member',
   aliases: [],
-  description: 'Gets CSC membership information.',
+  description: 'Get CSC membership information.',
```

Comment:
Get CSC membership information -> Get CSC membership information **for a user**?